src/atomate2/vasp/schemas/calculation.py

```diff
@@ -443,6 +449,10 @@ def from_vasp_outputs(
             magmoms = [m["tot"] for m in outcar.magnetization]
             structure.add_site_property("magmom", magmoms)
 
+        # Parse DOS properties
+        if hasattr(vasprun, "complete_dos"):
+            dosprop_dict = _get_band_props(vasprun.complete_dos, structure)
```

Comment:
Do you need to set a default value for dosprop_dict in case there isn't a complete dos?